src/JanusGraph.Net/Utils/LongEncoding.cs

```diff
@@ -26,11 +26,26 @@ namespace JanusGraph.Net.Utils
     /// <summary>
     ///     Utility class for encoding longs in strings, re-implemented from its Java equivalent.
     /// </summary>
-    internal static class LongEncoding
+    /// <remarks>
+    ///     JanusGraph encodes long IDs in the <see cref="RelationIdentifier"/> as strings.
+    /// </remarks>
+    public static class LongEncoding
     {
-        private const string BaseSymbols = "0123456789abcdefghijklmnopqrstuvwxyz";
+        /// <summary>
+        ///     The symbols used for the encoding.
+        /// </summary>
+        // ReSharper disable once MemberCanBePrivate.Global
+        public const string BaseSymbols = "0123456789abcdefghijklmnopqrstuvwxyz";
```

Comment:
Is there a reason to make this public?

Reply:
Yes, to be able to use it in the XML comment below:

```c#
/// <exception cref="ArgumentException">
///     Thrown if the string contains any invalid characters. Only <see cref="BaseSymbols"/> are allowed.
/// </exception>
```

and to make it available to users as it's an important part to know when using this class which symbols are used for the encoding.